s4/clients/s3.py

```diff
@@ -20,6 +22,11 @@
 
 S3Uri = collections.namedtuple("S3Uri", ["bucket", "key"])
 
+python_ignore = os.environ.get('PYTHONWARNINGS')
+if python_ignore == 'ignore:Unverified HTTPS request':
+    verifySSL = False
+else:
+    verifySSL = True
```

Comment:
could you explain why you would want this?

Reply:
> Behind my corporate firewall I can't connect using HTTPS because the corporate proxy exchange the encryptation keys to inspect the traffic (Security Here is High). 

Oh wow that's pretty horrible :| how do you connect to any website then? Most websites use HSTS nowadays which means they wont allow you to connect to them without an https connection.